patches/tock/08-remove-unused-ctap-module.patch

```diff
@@ -0,0 +1,588 @@
+From 26aa8751c4f0265c7389c9e2fb29f0e9bc44f7fa Mon Sep 17 00:00:00 2001
+From: Liam Murphy <liamjm@google.com>
+Date: Fri, 1 Apr 2022 11:52:11 +1100
+Subject: [PATCH] Remove unused ctap module (capsules/src/usb/ctap.rs)
+
+---
+ boards/components/src/ctap.rs | 127 -----------
+ boards/components/src/lib.rs  |   1 -
+ capsules/src/usb/ctap.rs      | 410 ----------------------------------
+ capsules/src/usb/mod.rs       |   1 -
+ 4 files changed, 539 deletions(-)
+ delete mode 100644 boards/components/src/ctap.rs
+ delete mode 100644 capsules/src/usb/ctap.rs
+
+diff --git a/boards/components/src/ctap.rs b/boards/components/src/ctap.rs
```

Comment:
For consistency with the other patches, everything before this line should not be present (as well as the signature at the end of the file). You can fix this by using the `./maintainers/patches` script. Simply run the following from your branch:

```
./maintainers/patches apply
./maintainers/patches save
./maintainers/patches restore
```